test/app-forms.spec.js

```diff
@@ -308,6 +312,22 @@ describe('forms that have caused bugs', () => {
     expect(result.errors).to.be.empty;
   });
 
+  it('#234 - fill datetime field', async () => {
+    const result = await harness.fillForm('bug_234', ['2023-01-01 1:22']);
+    expect(result.errors).to.be.empty;
+  });
+
+  it('#234 - fill datetime expected input format', () => {
```

Comment:
Maybe this can be a bit more descriptive, like `fill datetime expected correct input format` so intent is clear at a glance

Reply:
good idea. i've updated the names for all tests